### PR TITLE
Fixing the announcements & Adding support for GitLab

### PR DESCRIPTION
**Pull request by julianwastkn**
I've gone ahead and picked different colors for the different results that you can get from your installation. I wasn't sure about a few of these so go ahead and change them to whatever you like.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,7 +27,7 @@ module.exports = class InstallFromLink extends Plugin {
 			const {target} = this.props;
 			if (target.tagName.toLowerCase() === "a") {
 				const href = target.href
-				let match = href.match(/^https?:\/\/(www.)?github.com\/[\w-]+\/[\w-]+/)
```
Comment (cloudrac3r) [inline index.js]: Should be | instead of ||

File: index.js [hunk 2]
```diff
@@ -72,21 +71,21 @@ module.exports = class InstallFromLink extends Plugin {
 										try {
 											files = fs.readdirSync(pj(pluginsDir, repoName))
```
Comment (cloudrac3r) [inline index.js]: This should be red

**Review by cloudrac3r — CHANGES_REQUESTED** [review]
I hope I'm pressing the right buttons here

File: index.js [hunk 3]
```diff
@@ -72,21 +71,21 @@ module.exports = class InstallFromLink extends Plugin {
...
 											} else { // remount mysteriously failed
-												announce("The plugin was installed, but you'll need to press Ctrl-R to reload Discord before you can use it.")
+												announce("The plugin was installed, but you'll need to press Ctrl-R to reload Discord before you can use it.", "green")
 											}
 										} else { // no manifest
-											announce("That repo is not a Powercord plugin. It's been cloned anyway, but don't expect it to do anything.")
+											announce("That repo is not a Powercord plugin. It's been cloned anyway, but don't expect it to do anything.", "orange")
 										}
 									} else { // non-zero exit code
-										if (status) announce(status)
```
Comment (cloudrac3r) [inline index.js]: Unfinished sentence? Remove "do y" if you don't know what else to do with it.

File: manifest.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Install From Link",
-	"version": "1.1.0",
+	"version": "1.1.1",
```
Comment (cloudrac3r) [inline manifest.json]: This is more than a simple bug fix, so make it 1.2.0, please.

**Comment by cloudrac3r** [comment]
Hmm — looks like I didn't mark the right lines. Looks like I was only slightly off, so I'm sure you can figure it out. Let me know if you need help.

File: index.js [hunk 5]
```diff
@@ -28,7 +27,7 @@ module.exports = class InstallFromLink extends Plugin {
 			const {target} = this.props;
 			if (target.tagName.toLowerCase() === "a") {
 				const href = target.href
-				let match = href.match(/^https?:\/\/(www.)?github.com\/[\w-]+\/[\w-]+/)
+				let match = href.match(/^https?:\/\/(www.)?(github||gitlab).com\/[\w-]+\/[\w-]+/)
```
Comment (cloudrac3r) [inline index.js]: Should be | instead of ||

File: index.js [hunk 6]
```diff
@@ -72,21 +71,21 @@ module.exports = class InstallFromLink extends Plugin {
 										try {
 											files = fs.readdirSync(pj(pluginsDir, repoName))
 										} catch (e) { // readdirSync failed
-											return announce(`Cloning apparently succeeded, but the resulting installation folder was nowhere to be found. (Looking for ${pj(pluginsDir, repoName)})`) // just do this for now I guess
+											return announce(`Cloning apparently succeeded, but the resulting installation folder was nowhere to be found. (Looking for ${pj(pluginsDir, repoName)})`, "yellow") // just do this for now I guess
```
Comment (cloudrac3r) [inline index.js]: Make this red instead of yellow

File: index.js [hunk 7]
```diff
@@ -72,21 +71,21 @@ module.exports = class InstallFromLink extends Plugin {
...
 											}
 										} else { // no manifest
-											announce("That repo is not a Powercord plugin. It's been cloned anyway, but don't expect it to do anything.")
+											announce("That repo is not a Powercord plugin. It's been cloned anyway, but don't expect it to do anything.", "orange")
 										}
 									} else { // non-zero exit code
-										if (status) announce(status)
-										else announce(`The command mysteriously exited with a non-zero status code (${code}). Do y`)
+										if (status) announce(status, "red")
+										else announce(`The command mysteriously exited with a non-zero status code (${code}). Do y`, "red")
```
Comment (cloudrac3r) [inline index.js]: I don't know how that unfinished sentence got there. If you don't know what to do about it, just remove the "do y".

**Review by cloudrac3r — CHANGES_REQUESTED** [review]

**Comment by julianwastkn** [comment]
I think I've fixed all the things mentioned. Let me know if there's something that I missed.

**Review by cloudrac3r — APPROVED** [review]

**Pull request merged**